runtime/common/src/claims.rs

```diff
@@ -726,6 +726,7 @@ mod tests {
 
 	parameter_types! {
 		pub const MinVestedTransfer: u64 = 0;
+		pub const MinVestedTransfer: u32 = 28;
```

Comment:
Looks like you've got a duplicate definition here. You probably meant to name this `MaxVestingSchedules`?

Reply:
93554b3